Umbraco-Cloud/Deployment/Deploy-Settings/index.md

```diff
@@ -80,3 +80,19 @@ In order for your Cloud project to handle Forms data as content, you'll need to
 :::note
 This only needs to be done if your Cloud project was created **before** June 18th 2019. Make sure to [follow the instructions](../Umbraco-Forms-on-Cloud/#did-you-create-your-cloud-project-before-june-18th-2019) to succesfully change this setting on all environments.
 :::
+
+
+## ExportMemberGroups
+
+This is available in Deploy 3.4.0+ version and newer. 
+
+This setting is to be enabled only if you are using an external membership provider for your members. You will not want to export member groups that would no longer be managed by Umbraco but by an external membership provider.
```

Comment:
```suggestion
This setting is to be defined and set to false only if you are using an external membership provider for your members. You will not want to export member groups that would no longer be managed by Umbraco but by an external membership provider.
```

Good spot, what I was trying to say was more like if this setting exists or is present, but I can understand the confusion.

Let me know what you think of this @sofietoft

Reply:
Super! Makes sense 💪 Thanks for updating it!